component/Page/Profile/index.tsx

```diff
@@ -87,7 +87,8 @@ export default function ProfilePage(props: PageProps) {
         let count = linesRegex.length;
         const lines = count
 
-        const numLines = lines <= 2 ? lines : 2;
+        // const numLines = lines <= 4 ? lines : 4;
+        const numLines = 4
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a dynamic number of lines to a fixed number might limit the flexibility of the component. If this is intentional and you want to always display 4 lines regardless of the content, then it's fine. However, if you want to maintain the previous behavior (displaying up to 2 lines), you should revert this change.
```diff
- const numLines = 4
+ const numLines = lines <= 2 ? lines : 2;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->